filesystem/luks.ksy

```diff
@@ -0,0 +1,72 @@
+meta:
+  id: luks
+  title: Linux Unified Key Setup
+  license: CC0-1.0
+  endian: be
+  encoding: ASCII
+doc: |
+  Linux Unified Key Setup (LUKS) is a format specification for storing disk
+  encryption parameters and up to 8 user keys (which can unlock the master key).
+doc-ref: https://gitlab.com/cryptsetup/cryptsetup/wikis/LUKS-standard/on-disk-format.pdf
+seq:
+  - id: partition_header
+    type: partition_header
+types:
+  partition_header:
+    seq:
+      - id: magic
+        contents: [0x4C, 0x55, 0x4B, 0x53, 0xBA, 0xBE]
+      - id: version
+        contents: [0x00, 0x01]
+      - id: cipher_name_specification
+        type: str
+        size: 32
+      - id: cipher_mode_specification
+        type: str
+        size: 32
+      - id: hash_specification
+        type: str
+        size: 32
+      - id: payload_offset
+        type: u4
```

Comment:
If we'll stick to [style guide](http://doc.kaitai.io/ksy_style_guide.html#attr-id), it's recommended to use `ofs_payload` for attributes like this.